batch.go

```diff
@@ -121,11 +122,8 @@ func (b *Batch) Get(key []byte) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-
 	record := decodeLogRecord(chunk)
-	if record.Type == LogRecordDeleted {
```

Comment:
we can add an assertion here if we assume that the Type will never be `LogRecordDeleted`.
eg. if record.Type == LogRecordDeleted { panic("some prompt messages...")}